docs/data/base/components/select/UnstyledSelectObjectValuesForm.tsx

```diff
@@ -35,12 +36,12 @@ const StyledButton = styled('button')(
   font-family: IBM Plex Sans, sans-serif;
   font-size: 0.875rem;
   box-sizing: border-box;
+  display: block;
   min-height: calc(1.5em + 22px);
   min-width: 320px;
   background: ${theme.palette.mode === 'dark' ? grey[900] : '#fff'};
   border: 1px solid ${theme.palette.mode === 'dark' ? grey[800] : grey[300]};
   border-radius: 0.75em;
-  margin: 0.5em;
```

Comment:
Primitives like a select shouldn't have a margin, the parent component usually takes care of it.